parcellab/common/templates/_keda.tpl

```diff
@@ -10,6 +10,7 @@
 {{- define "keda.hpa" -}}
 {{- $service := default dict .service -}}
 {{- $componentValues := (merge (deepCopy .) (dict "component" $service.name)) -}}
+{{- $scaleOnInFlight := (ternary $service.keda.scaleOnInFlight false (eq (typeOf $service.keda.scaleOnInFlight) "bool" )) -}}
```

Comment:
not sure I get it, this will evaluate to `true` if `scaleOnInFlight` is a bool?

Reply:
That's right. If $service.keda.scaleOnInFlight is a bool, then $scaleOnInFlight will have that value. If not, then $scaleOnInFlight will be false.

In other works, any scaleOnInFlight value in the values(-external).yaml file that is not `true` will be changed to `false`

Reply:
ah, the predicate for ternary is the last parameter, got it.